### ISSUE TITLE
Social Login Facebook configuration fields missing in Admin

### ISSUE THREAD

**Issue by macfisher** [mageplaza/magento-2-social-login#53]
### Preconditions

1. Magento:  2.1.6
2. PHP: 7.0.16-1
3. MySQL: 14.14
4. 

### Steps to reproduce

1. Login to Admin
2. Go to Mageplaza->Social Login
3. See General, Google, Twitter, LinkedIn, Instagram, Yahoo, Github, Vkontakte, Foursquare and Live, but there is no Facebook configuration area.

### Expected result

1. Should see a Facebook area as shown in the Official Documentation. I need to get the OAuth Return URI from this field.

### Actual result

1. No Facebook configuration fields at all.

**Comment by luongvantam59** [comment]
Hi,
It's working well on my local machine, Facebook configuration fields show up right on Google configuration fields.You can see it at this => [http://prntscr.com/gbfypt](http://prntscr.com/gbfypt)

**Comment by macfisher** [comment]
@MaxxLuong Thank you for the reply. Your screen shot definitely showcases the expected result, and in my case the expected result was not true. I do not need this resolved since I just added my API key and API secret directly to the `core_config_data` table and I took a wild guess and made my own OAuth Redirect URI.

I just felt it necessary to create an issue to give you a chance to maybe fix this before more people are able to reproduce the issue.
